src/go/bootstrap/static/bootstrap_test.go

```diff
@@ -95,61 +95,67 @@ func TestServiceToBootstrapConfig(t *testing.T) {
 			serviceConfigPath: platform.GetFilePath(platform.GrpcEchoServiceConfig),
 			envoyConfigPath:   platform.GetFilePath(platform.GrpcEchoEnvoyConfig),
 		},
+		{
+			desc: "envoy config for sidecar with x-google-backend",
+			opt_mod: func(opt *options.ConfigGeneratorOptions) {
+				opt.AdminPort = 0
+				opt.BackendAddress = "http://127.0.0.1:8082"
+				opt.DisableTracing = true
+				opt.SkipServiceControlFilter = true
+			},
+			serviceConfigPath: platform.GetFilePath(platform.SbServiceConfig),
+			envoyConfigPath:   platform.GetFilePath(platform.SbEnvoyConfig),
+		},
 	}
 
 	for _, tc := range testData {
-		configBytes, err := ioutil.ReadFile(tc.serviceConfigPath)
-		if err != nil {
-			t.Errorf("ReadFile failed, got %v", err)
-			continue
-		}
-		unmarshaler := &jsonpb.Unmarshaler{
-			AnyResolver:        util.Resolver,
-			AllowUnknownFields: false,
-		}
+		t.Run(tc.desc, func(t *testing.T) {
+			configBytes, err := ioutil.ReadFile(tc.serviceConfigPath)
+			if err != nil {
+				t.Errorf("ReadFile failed, got %v", err)
+			}
+
+			unmarshaler := &jsonpb.Unmarshaler{
+				AnyResolver:        util.Resolver,
+				AllowUnknownFields: false,
+			}
 
-		var s confpb.Service
-		if err := unmarshaler.Unmarshal(bytes.NewBuffer(configBytes), &s); err != nil {
-			t.Errorf("Unmarshal() returned error %v, want nil", err)
-			continue
-		}
+			var s confpb.Service
+			if err := unmarshaler.Unmarshal(bytes.NewBuffer(configBytes), &s); err != nil {
+				t.Errorf("Unmarshal() returned error %v, want nil", err)
+			}
 
-		opts := flags.EnvoyConfigOptionsFromFlags()
-		tc.opt_mod(&opts)
+			opts := flags.EnvoyConfigOptionsFromFlags()
```

Comment:
NIT: different indent?

Reply:
I moved everything into `t.Run`, everything should be indented. I don't think it's a different indent, i ran `make format`.

Reply:
SG